flink-kubernetes/src/main/java/org/apache/flink/kubernetes/kubeclient/parameters/KubernetesJobManagerParameters.java

```diff
@@ -63,6 +65,12 @@ public KubernetesJobManagerParameters(Configuration flinkConfig, ClusterSpecific
 		return getPrefixedEnvironments(ResourceManagerOptions.CONTAINERIZED_MASTER_ENV_PREFIX);
 	}
 
+	@Nullable
```

Comment:
This should either go to the interface `KubernetesParameters` or preferrably we return an empty map if it has not been set.

Reply:
Done via [29a819d](https://github.com/apache/flink/pull/10973/commits/29a819da0513b97fb78e586ef884bfa27b4eeb89)